src/index.html

```diff
@@ -1,14 +1,84 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>flexbox</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a class="a" href="index.html">
+        <img
+          class="logo-img"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="ul-decor">
+          <li class="li-set">
+            <a class="is-active nav-set text-decoration"
+             href="#apple">
```

Comment:
<img width="1069" alt="Attributes formatting" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/0e5ab7a5-12e9-473d-a245-e3aacca1f8ac">

The same in all other places